docs/source/installation/win.rst

```diff
@@ -1,13 +1,73 @@
 Windows
 =======
 
-First, you need to install Chocolatey, which is a package manager for Windows
-systems.  Please refer to `this link <https://chocolatey.org/install>`_ for
+There are two simple ways to download manim's depedencies, using the popular package
+managers `Scoop <https://shoop.sh>`_ and `Chocolatey <https://chocolatey.org/install>`_
+
+.. _scoop:
+
+Installing using Scoop
+**********************
+
+First you need to install Scoop, which is a command-line installer for Windows
+systems. Please refer to `this link
+<https://scoop-docs.now.sh/docs/getting-started/Quick-Start.html>`_ for
 instructions.
 
+While a manifest for manim doesn't currently exist, it is sufficient to install the dependencies
+via scoop and manim itself via ``pip``.
+
+After installing Scoop, add these "buckets" as we'll need to install things from them later:
+
+.. code-block:: powershell
+
+      scoop bucket add extras
+      scoop bucket add r-bucket https://github.com/cderv/r-bucket.git
+
+FFmpeg installation
+-------------------
+1. Run ``scoop install ffmpeg``
+2. Check whether ffmpeg has been properly installed by running ``ffmpeg``.
+
+LaTeX Installation
+------------------
+There are two ways of installing a LaTeX distribution that will be covered here:
+
+1. using MikTeX (approx. 2GB, takes a while to install, but provides "on-the-fly"
+package installation)
```

Comment:
```suggestion
package installation but is recommended for beginners)
```
The reason I wrote this is because most of beginners might not know where and how to download the LaTeX package and sometimes could be really cumbersome.

Reply:
I agree, but two `but`s in a single clause seem weird imo. Maybe `....package installation, recommended for beginners)`?

Reply:
you should possibly tell this for chocolatey install section also.

Reply:
Yeah changed it and I agree with @naveen521kk.